### PR TITLE
[Webbrowser] Add 'controlsColor' to OpenBrowserOptions type

### LINKED ISSUE #4910: [docs] controlsColor and collapseToolbar are in the Doc but not available in expo-web-browser
In the Doc, it says:
https://docs.expo.io/versions/latest/sdk/webbrowser/
```
controlsColor (optional) (string) -- (iOS only) tint color for controls in SKSafariViewController in #AARRGGBB or #RRGGBB format.

collapseToolbar (optional) (boolean) -- a boolean determining whether the toolbar should be hiding when a user scrolls the website
```
But it is not available in the code:
https://github.com/expo/expo/blob/master/packages/expo-web-browser/src/WebBrowser.types.ts#L5

Is it something missing in the code or something wrong in the Doc?

### PR DESCRIPTION
**Pull request by cruzach**
# Why

Closes #4910 

`controlsColor` is an [optional (iOS only) argument according to the docs](https://docs.expo.io/versions/latest/sdk/webbrowser/#webbrowseropenbrowserasyncurl), and is used 
here https://github.com/expo/expo/blob/master/packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m#L9

### CODE REVIEW

File: packages/expo-web-browser/src/WebBrowser.types.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export type OpenBrowserOptions = {
   browserPackage?: string;
   enableBarCollapsing?: boolean;
   showTitle?: boolean;
+  controlsColor?: string;
```
Comment (tsapeta) [inline packages/expo-web-browser/src/WebBrowser.types.ts]:
```suggestion
  controlsColor?: string; // iOS only
```
We usually add a comment if the property is available only on specific platforms 😉 Don't forget to rebuild JS files after this change.

**Review by tsapeta — APPROVED** [review]

File: packages/expo-web-browser/src/WebBrowser.types.ts (hunk 1 above)
Comment (sjchmiela) [inline packages/expo-web-browser/src/WebBrowser.types.ts]:
```suggestion
  /** iOS only */
  controlsColor?: string;
```
```suggestion
  /*
  * iOS only
  */
  controlsColor?: string;
```

Would this make the comment show up in a helpful popup in VSCode? Do you remember @tsapeta when we researched [this](https://swmansion.slack.com/archives/D0XMLA4RJ/p1560782613060000)?

Comment (tsapeta) [reply packages/expo-web-browser/src/WebBrowser.types.ts]: oh yeah, that's right - the first one (`/** iOS only */`) will popup in VSCode

Comment (cruzach) [reply packages/expo-web-browser/src/WebBrowser.types.ts]: Got it, thanks for pointing that out!

**Pull request merged**